.gitlab-ci.yml

```diff
@@ -295,7 +295,7 @@ buildtest:typescript-apis:
     POSTGRES_PASSWORD: ""
     OPA_URL: "http://docker:8181/"
   script:
-    - cd deploy/helm/magda-core/charts/opa
+    - cd deploy/helm/internal-charts/opa
     - docker-compose up -d
     - cd ../../../../..
```

Comment:
the CI pipeline fails at this moment.
It seems caused by  line `cd ../../../../..`.
Since we changed the path, we should also adjust this line

Reply:
Adjusted